components/tokens/custom-spectrum/custom-vars.css

```diff
@@ -18,6 +18,7 @@ governing permissions and limitations under the License.
 .spectrum {
   --spectrum-animation-duration-100: 130ms;
   --spectrum-animation-duration-200: 160ms;
+  --spectrum-animation-duration-500: 250ms;
```

Comment:
We will have to do some refactoring once we have those tokens in the package. 
We used those animation tokens very often.